code/controllers/configuration.dm

```diff
@@ -347,6 +350,10 @@
 						world.log = newlog
 				if("autoconvert_notes")
 					config.autoconvert_notes = 1
+				if("announce_admin_logout")
+					config.announce_admin_logout = 1

```

Comment:
config.announce_admin_logout = 1 is already set to 1, so this has no effect, if it wasn't present it would just still be at 1